spacy/lang/nl/lex_attrs.py

```diff
@@ -3,38 +3,198 @@
 
 from ...attrs import LIKE_NUM
 
+import regex
 
-_num_words = set("""
-nul een één twee drie vier vijf zes zeven acht negen tien elf twaalf dertien
-veertien twintig dertig veertig vijftig zestig zeventig tachtig negentig honderd
-duizend miljoen miljard biljoen biljard triljoen triljard
-""".split())
+VIJF = 'vijf'
+ZES = 'zes'
+ZEVEN = 'zeven'
+ACHT = 'acht'
+NEGEN = 'negen'
+TIEN = 'tien'
+HONDERD = "honderd"
+DUIZEND = "duizend"
 
-_ordinal_words = set("""
-eerste tweede derde vierde vijfde zesde zevende achtste negende tiende elfde
-twaalfde dertiende veertiende twintigste dertigste veertigste vijftigste
-zestigste zeventigste tachtigste negentigste honderdste duizendste miljoenste
-miljardste biljoenste biljardste triljoenste triljardste
-""".split())
+wrong_ordinals_re = regex.compile(r"""(?x)
+.*(?P<wrong>nulste|eende|tweeste|driede|vierste|vijfste|zeste|zesste|achtde|tienste|honderdde|duizendde)
```

Comment:
In my dialect, we may actually say "tienste" instead of "tiende". Also found a vague reference here: https://www.encyclo.nl/begrip/tienste 
So maybe excplicitely including it as wrong is not necessary?

Reply:
They are verbose regular expressions, spaces are ignored, unless added as explicit r"\s"

What about eenste and drieste? :-) And I removed zevenste and negenste because they are valid in my own dialect, but I'm not convinced that was the right thing to do. Once decided whether to accept them or not, I should definitely add them to the test cases.

I noticed that in another pull request there was a move away from regex to re. Is it better to use re when possible? It shouldn't pose a problem for Dutch.

Reply:
I did it the good old fashioned Belgian way: a compromise. I added a parameter. Those who want to adhere strictly to Dutch AN spelling, can set strict_AN_spelling to True.
I removed some of the edge cases from the unit tests.
Thanks for your review, it's really appreciated!